automatic/slack/tools/chocolateyBeforeModify.ps1

```diff
@@ -1 +1,10 @@
-﻿Get-Process "slack" -ErrorAction SilentlyContinue | Stop-Process -Force
+﻿$SlackPath = Join-Path -Path $Env:ProgramFiles -ChildPath 'Slack\slack.exe'
+$InstalledVersion = (Get-ItemProperty -Path $SlackPath -ErrorAction:SilentlyContinue).VersionInfo.ProductVersion
+$SlackOutdated = [Version]$($Env:ChocolateyPackageVersion) -gt [Version]$InstalledVersion
+
+# Only Attempt to kill the slack process if the existing version is the same or newer than the package version
+if ($SlackOutdated)
```

Comment:
This will break a forced install of the same Slack version.

Reply:
I have removed this file entirely, to avoid duplicating the logic (present/outdated/forced) in two different places.